.azure-pipelines/steps/download-artifact.yml

```diff
@@ -0,0 +1,21 @@
+# .azure-pipelines/steps/download-artifact.yml
+parameters:
+  artifact: ''
+  path: ''
+  patterns: ''
+  displayName: ''
```

Comment:
Meh, I think we can probably just remove this tbh, the artifact name tells you what it is - it was mostly there because if you don't provide it, it just says "Download artifact" instead